master/internal/db/postgres_rp_workspace_bindings.go

```diff
@@ -170,3 +170,50 @@ func OverwriteRPWorkspaceBindings(ctx context.Context,
 	err = AddRPWorkspaceBindings(ctx, workspaceIds, poolName, resourcePools)
 	return err
 }
+
+// GetUnboundRPs get unbound resource pools.
+func GetUnboundRPs(
+	ctx context.Context, resourcePools []config.ResourcePoolConfig,
+) ([]string, error) {
+	var boundResourcePools []string
+	_, err := Bun().NewSelect().Model(&boundResourcePools).Column("name").Distinct().Exec(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Store boundRPs into a map for efficiency.
```

Comment:
nit: I don't think you need to comment here - I think it's pretty clear why you are using a map.